maintenance-mode/nginx.conf

```diff
@@ -0,0 +1,23 @@
+user nonrootuser;
```

Comment:
I have a doubt that it would work on Openshift.
Did you do research on the official nginx docker hub?